source/documentation/reference/operational-processes.md

```diff
@@ -176,3 +158,8 @@ We would like to measure:
 
 We do not currently have a process in place for measuring these but we are working on it.
 
+#### Our TD documentation
```

Comment:
Not sure why this exists here?

Reply:
I guess we still need to point to it while TD lives even if its just for new CP team members? or point to to if users ask? We could move it but i think it needs to live somewhere i think?

Reply:
Yes, but this reference should appear in the docs section that was removed by mistake. See comments in my other review.

Reply:
ok, sorted. Copy and paste error with wrong section. And changed over the order in the documentation section